prisma-fmt-wasm/default.nix

```diff
@@ -0,0 +1,65 @@
+{ crane, nixpkgs, rust-overlay, system, src }:
+
+let
+  overlays = [
+    rust-overlay.overlays.default
+    (self: super:
+      let toolchain = super.rust-bin.fromRustupToolchainFile ./rust-toolchain.toml; in
+      { cargo = toolchain; rustc = toolchain; })
+  ];
+  pkgs = import nixpkgs { inherit system overlays; };
+  craneLib = crane.mkLib pkgs;
+
+  inherit (pkgs) jq nodejs coreutils rustPlatform wasm-bindgen-cli;
+  inherit (builtins) readFile replaceStrings;
+in
+rec {
+  packages = {
+    prisma-fmt-wasm = craneLib.buildPackage {
+      pname = "prisma-fmt-wasm";
+      version = "0.1.0";
+
+      nativeBuildInputs = with pkgs; [ git wasm-bindgen-cli ];
+
+      cargoBuildCommand = "cargo build --release --target=wasm32-unknown-unknown --manifest-path=prisma-fmt-wasm/Cargo.toml";
+      cargoCheckCommand = "cargo check --target=wasm32-unknown-unknown --manifest-path=prisma-fmt-wasm/Cargo.toml";
+      doCheck = false; # do not run cargo test
+      cargoArtifacts = null; # do not cache dependencies
+
+      installPhase = readFile ./scripts/install.sh;
+
+      inherit src;
+    };
+
+    # Takes a package version as its single argument, and produces
+    # prisma-fmt-wasm with the right package.json at /tmp/package. This is used
```

Comment:
Wrong: not `/tmp/package`. It prints the path to a temporary directory where it wrote the package.